package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "test-ui-lib",
-  "version": "0.0.4",
-  "description": "React UI component lib",
-  "author": "martinmcneela",
+  "name": "ui-component-lib",
+  "version": "0.0.1",
+  "description": "React UI component lib for edozo apps and sites",
+  "author": "edozo",
```

Comment:
I was a wee bit tentative about putting edozo in here, just to make it less searchable. But I guess it has to be for the author and repo?

Reply:
It would essentially just be garbage data if we change it. Is there a reason you particularly wanted to hide it?

Reply:
Mainly because we haven't discussed open-source with the business.